R/utilities-simulation.R

```diff
@@ -425,6 +425,22 @@ getAllParametersForSensitivityAnalysisMatching <- function(paths, simulation) {
   )
 }
 
+#' Get the paths of all state variable quantities of the simulation
+#'
+#' @param simulation \code{Simulation} object
+#' @details List of paths of all molecules in all compartments and all parameters that are
+#' state variables.
+#'
+#' @return A list of paths
+#' @export
+getAllStateVariablesPaths <- function(simulation) {
+  validateIsOfType(simulation, type = "Simulation")
+  allMoleculesPaths <- getAllEntityPathsIn(container = simulation, entityType = Molecule)

```

Comment:
getAllMoleculePathsIn instead. Since we have a convenience method for this, why not use it?

Reply:
Done